deps/README.md

```diff
@@ -100,8 +100,8 @@ Hdr_Histogram
 ---
 
 Updated source can be found here: https://github.com/HdrHistogram/HdrHistogram_c
-We use a customized version 0.11.5
-1. Compare all changes under /hdr_histogram directory to version 0.11.5
```

Comment:
so just to double check.
you did follow this procedure, right?
(didn't just copy the changes from you other PR)

Reply:
That's the exact procedure I did (wouldn't have compiled otherwise)